lib/hacluster.pm

```diff
@@ -1616,10 +1617,17 @@ asserting anything on the screen, clear it, and then select it normally.
 =cut
 
 sub prepare_console_for_fencing {
-    select_console 'root-console', await_console => 0;
+    if (is_pvm) {
+        record_info 'TEAM-10830';
```

Comment:
Can you please replace ticket number with something descriptive? 
It might be useful especially for people without Jira access.